portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/site.vm

```diff
@@ -27,6 +27,17 @@
 
     <body class="Mrphs-portalBody overflow-hidden $!{loggedInClass}" data-bs-no-jquery="true">
 
+        <script>
```

Comment:
Could this be a small script file? Then it can be cached. Else this needs to be loaded every page load?